src/components/atoms/Button/Button.styles.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button`
+  border: none;
+  border-radius: 25px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 15px;
+  font-family: 'Spartan', sans-serif;
+  transition: 0.3s;
+
+  padding: ${({ role }) => (role === 'buttonOne' ? '5px 25px 5px 8px' : '15px 25px')};
+
+  width: ${({ role }) => (role === 'buttonSix' ? '100%' : 'auto')};
+
+  background: ${({ role }) => {
+    switch (role) {
+      case 'buttonOne':
+        return '#7C5DFA';
+      case 'buttonTwo':
+        return '#7C5DFA';
+      case 'buttonThreeLight':
+        return '#F9FAFE';
+      case 'buttonThreeDark':
+        return '#252945';
+      case 'buttonFourLight':
+        return '#373B53';
+      case 'buttonFourDark':
+        return '#373B53';
+      case 'buttonFive':
+        return '#EC5757';
+      case 'buttonSix':
+        return '#F9FAFE';
+
+      default:
+        return '#FFFFFF';
+    }
```

Comment:
Note: All of the color values can be taken from the utils once they are merged, just like above. However, for the time being it's OK :)